public/style/main.css

```diff
@@ -128,6 +164,127 @@ nav div.section {
   overflow-y       : hidden;
   overflow-x       : hidden;
 }
+
+/* BOTTOM-UP CONTROL PANEL MODULES */
+
+.controlModules {
+  font-size: 1.2em;
```

Comment:
the fact that there are so many classes only changing em size indicates there is some severe issues with font size consistency